README.md

```diff
@@ -731,11 +731,15 @@ This is useful if you do not want to explicitly set up your registry
 configuration in all of your package.json files individually when e.g. using
 private registries.
 
-#### --skip-temp-tag
+#### --skip-temp-tag (DEPRECATED)
```

Comment:
I think we can just replace `--skip-temp-tag` with `--use-temp-tag`, since we're breaking the default behaviour anyway. Even if `--skip-temp-tag` is passed, it won't matter because that's what we'll be doing anyway.

The description of the flag would be similar to the existing docs, but slightly tweaked:
----
#### --use-temp-tag

When passed, this flag will alter the default publish process by first publishing all changed packages to a temporary dist-tag (`lerna-temp`) and then moving the new version(s) to the default [dist-tag](https://docs.npmjs.com/cli/dist-tag) (`latest`).

This is not generally necessary, as Lerna will publish packages in topological order (all dependencies before dependents) by default.